### PR TITLE
docs(concepts): fix link

### PR DESCRIPTION
**Pull request by hitsuji-no-shippo**
_describe your changes..._

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/

### CODE REVIEW

File: src/content/concepts/module-resolution.md [hunk 1]
```diff
@@ -55,18 +55,18 @@ import 'module';
...
 
-- If the folder contains a `package.json` file, then fields specified in [`resolve.mainFields`](/configuration/resolve/#resolve-mainfields) configuration option are looked up in order, and the first such field in `package.json` determines the file path.
-- If there is no `package.json` or if the main fields do not return a valid path, file names specified in the [`resolve.mainFiles`](/configuration/resolve/#resolve-mainfiles) configuration option are looked for in order, to see if a matching filename exists in the imported/required directory .
+- If the folder contains a `package.json` file, then fields specified in [`resolve.mainFields`](/configuration/resolve/#resolvemainfields) configuration option are looked up in order, and the first such field in `package.json` determines the file path.
+- If there is no `package.json` or if the main fields do not return a valid path, file names specified in the [`resolve.mainFiles`](/configuration/resolve/#resolvemainfiles) configuration option are looked for in order, to see if a matching filename exists in the imported/required directory .
 - The file extension is then resolved in a similar way using the `resolve.extensions` option.
```
Comment (hemal7735) [inline src/content/concepts/module-resolution.md]: can you add link for `resolve.extensions` ->  `/configuration/resolve/#resolveextensions`

**Comment by hemal7735** [comment]
hi @hitsuji-no-shippo , please sign the CLA to move forward this PR.

**Comment by hitsuji-no-shippo** [comment]
I created this sentence using google translation because I can not speak English. I am sorry if this sentence is difficult to read.

@hemal7735 CLA's signature was invalidated because the commit e-mail address was different, so I fixed it. That's the reason for using force-pushed. I've also added a link.

**Comment by montogeek** [comment]
@EugeneHlushko Why is this not getting caught in the build? Did we disable it?
cc @Munter

**Comment by EugeneHlushko** [comment]
@montogeek this is not getting caught because the url responds with 200... Its the anchor that is unstable for some reason

**Comment by Munter** [comment]
@montogeek @EugeneHlushko Hyperlink is supposed to check the correctness of fragment identifier links. Hyperlinks test suite has test cases for this exact situation. I can't explain why it doesn't discover these ones without doing a deeper debugging. My first intuition is that maybe this page is not on a discoverable part of the dependency graph, so none of its outgoing links are checked at all. But I haven't had time to spin up an environment and have a look at it yet

**Comment by rishabh3112** [comment]
@hitsuji-no-shippo there are some merge conflicts, rest lgtm :)

**Comment by Munter** [comment]
@montogeek @EugeneHlushko Turns out there is indeed a bug in Hyperlink, which I have corrected on the current `master` branch. I'll make a release next time I have a bit of time and will start a branch to fix broken internal fragment links along with the update

**Comment by Munter** [comment]
@montogeek 	@EugeneHlushko hyperlink@4.1.2 fixes the missing fragment check

**Comment by hitsuji-no-shippo** [comment]
@rishabh3112 Thank you for telling me about conflicts. Just noticed today. I'm sorry for being late to notice that you said in July.

**Pull request merged**